package.json

```
@@ -0,0 +1,13 @@
+{
+  "dependencies": {
+    "vue": "^3.5.13"
+  },
+  "devDependencies": {
+    "vitepress": "^1.5.0"
+  },
+  "scripts": {
+    "docs:dev": "vitepress dev",
+    "docs:build": "vitepress build",
+    "docs:preview": "vitepress preview"
```

Remove docs prefix as the repository is only docs, this will help later down the line with StackBlitz for example.

```suggestion
    "dev": "vitepress dev",
    "build": "vitepress build",
    "preview": "vitepress preview"
```